File: Data Structures & Algorithms/longest-repeating-substring-with-replacement/submission-0.py
class Solution:
    def characterReplacement(self, s: str, k: int) -> int:
        stringSet = set(s)
        result = 0
        for c in stringSet:
            l = count = 0
            for r in range(len(s)):
                if s[r] == c:
                    count += 1
                while (r - l + 1) - count > k:
                    if s[l] == c:
                        count -= 1
                    l += 1

                result = max(result, r - l + 1)
        return result

File: Data Structures & Algorithms/max-consecutive-ones/submission-0.cpp
class Solution {
public:
    int findMaxConsecutiveOnes(vector<int>& nums) {
        int res = 0;
        int count = 0;
        for (int i = 0; i < nums.size(); i++)
        {
            count = nums[i] ? count + 1 : 0;
            res = max(res, count);
        }
        return res;
    }
};